FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -338,4 +358,26 @@ extension ComposeMessageService {
 
         return SendableMsgBody(text: text, html: html)
     }
+
+    private func isMessagePasswordStrong(pwd: String, isFesUsed: Bool) -> Bool {
+        let minLength = 8
+
+        guard isFesUsed else {
+            // consumers - just 8 chars requirement
+            return pwd.count >= minLength
+        }
```

Comment:
We don't support this scenario on iOS yet - the only way to send a password-protected message is with FES. It's ok to add this code but it should come with a note.